### PR TITLE
Added instructions for CocoaPod users ✅

### PR DESCRIPTION
**Pull request by trackq**
Added instructions to make the library useable for CocoaPod users ✌️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ React Native wrapper for Intercom.io. Based off of [intercom-cordova](https://gi
     ```bash
     react-native link react-native-intercom
     ```
+    
+    N.B. If CocoaPods was used for the installation of the Intercom library. Remove the link to node_modules from the Podfile and run `pod install again`. This line needs to be removed: `pod 'react-native-intercom', :path => '../node_modules/react-native-intercom'`
```
Comment (EvanGlazer) [inline README.md]: This needs to be removed alongside adding RNIntercom.xcodeproj.

**Comment by radiosilence** [comment]
Do we need to do anything to our Podfile if we're using 0.60+ ? It looks like there's a .podspec file so perhaps RN will autolink it?

**Comment by trackq** [comment]
> Do we need to do anything to our Podfile if we're using 0.60+ ? It looks like there's a .podspec file so perhaps RN will autolink it?

Not that I can remember.

**Comment by hamoudaarfaoui** [comment]
@trackq yes `autolink` would work, you don't have to update your podfile ;)

File: README.md (hunk 1 above)
Comment (TPXP) [inline README.md]: Looks like you have a small typo here ;-)
```suggestion
    N.B. If CocoaPods was used for the installation of the Intercom library. Remove the link to node_modules from the Podfile and run `pod install` again. This line needs to be removed: `pod 'react-native-intercom', :path => '../node_modules/react-native-intercom'`
```

**Pull request closed without merging**